infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -145,17 +259,19 @@ def restore_database(backup_fpath):
 
 
 def main():
+    print("System args:", sys.argv)
     print("Running database-backup process with subcommand", sys.argv[1])
     if sys.argv[1] == "backup":
-        backup_database(make_backup_fpath())
+        s3_backup_object_name = make_backup_object_name_from_datetime()
```

Comment:
Try to use type annotations for variables, especially when the value is returned from a function

```suggestion
        s3_backup_object_name: str = make_backup_object_name_from_datetime()
```

Reply:
separate pr with running a more recent version of python